debian/cloud-init.postinst

```diff
@@ -327,6 +327,22 @@ fix_lp1889555() {
     db_set grub-pc/install_devices_empty "false"
 }
 
+change_cloud_init_output_permissions() {
```

Comment:
A nit, but `output_log` makes it clearer what this is about when examining the list of all-the-things below: "cloud init output permissions" could mean a lot of things, "cloud init output log permissions" feels a little clearer.